ckcp/gitops.sh

```diff
@@ -143,15 +140,15 @@ install_openshift_gitops() {
   # Log into ArgoCD
   echo -n "  - ArgoCD Login: "
   local argocd_password
-  argocd_password="$(oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath="{.data.admin\.password}" | base64 --decode --ignore-garbage)"
+  argocd_password="$(oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath="{.data.admin\.password}" | base64 --decode)"
```

Comment:
I'm actually not sure why we are using `--ignore-garbage` here. It's not compatible with Mac OS `base64`